packages/backend-app-api/api-report.md

```diff
@@ -48,7 +48,7 @@ export interface Backend {
 }
 
 // @public (undocumented)
-export const cacheServiceFactory: () => ServiceFactory<PluginCacheManager>;
+export const cacheServiceFactory: () => ServiceFactory<CacheClient>;
```

Comment:
Hm. It's unfortunate to not be able to call this type `CacheService`.

Would it be possible to not change the `CacheClient`, and instead do essentially

```ts
export interface CacheService {
  get(); set(); delete(); withOptions();
  /** @deprecated use withOptions instead */
  getClient();
}
/** @deprecated */
export interface CacheClient {...}
```

Reply:
Ahh yeah I see, should have read the rest of the code more carefully.